ui/onboarding/Login/AccountSelection.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.3
+import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.11
+import QtQuick.Window 2.11
+import QtQuick.Dialogs 1.3
+import "../../shared"
+import "../../imports"
+
+Item {
+    property alias btnGenKey: btnGenKey
+//    property alias selectedIndex: selecte
+
+    id: wizardStep1
+    property int selectedIndex: 0
+    Layout.fillHeight: true
+    Layout.fillWidth: true
+
+    Text {
+        id: title
+        text: "Login"
+        font.pointSize: 36
+        anchors.top: parent.top
+        anchors.topMargin: 20
+        anchors.horizontalCenter: parent.horizontalCenter
+    }
+
+    ButtonGroup {
+        id: accountGroup
+    }
+
+    AccountList {
+        id: accountList
+        accounts: loginModel
+    }
+
+    Item {
+        id: footer
+        width: btnGenKey.width + selectBtn.width + Theme.padding
+        height: btnGenKey.height
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: Theme.padding
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        StyledButton {
+            id: btnGenKey
+            label: "Generate new account"
```

Comment:
```suggestion
            label: qsTr("Generate new account")
```

Reply:
if the property is for GenKey.qml, then will do in a different PR when dealing with GenKey